haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -527,6 +527,7 @@ data AllowXTZParam
   | Transfer_contract_tokens TransferContractTokensParam
   | Transfer_ownership TransferOwnershipParam
   | Accept_ownership ()
+  | Default ()
```

Comment:
This commit, e4149eccb8174f87ab4d8e5e97c2d0d452617349, lacks the `Problem`/`Solution` in the commit message